devtools/server/actors/replay/moz.build

```diff
@@ -9,9 +9,11 @@ DIRS += [
 ]
 
 DevToolsModules(
+    'apiServer.js',
```

Comment:
It's a nit, but this filename is using camelCase when everything else uses kebab-case.